enterprise/utils.py

```diff
@@ -123,3 +124,28 @@ def patch_mako_lookup():
     # Both add an item to the setting AND insert the lookup for immediate use
     settings.MAKO_TEMPLATES['main'].insert(0, template_location)
     add_lookup('main', template_location)
+
+
+def get_catalog_admin_url_template():
```

Comment:
Instead of getting the template, how about `get_catalog_admin_url(catalog_id)`?  It could call this method internally if we want to split out the template piece.